tuxemon/ai.py

```diff
@@ -81,12 +95,49 @@ def make_decision(
 class RandomAI(AI):
     """AI will use random technique against random opponent."""
 
-    def make_decision(
+    def make_decision_trainer(
+        self,
+        user: NPC,
+        monster: Monster,
+        opponents: Sequence[Monster],
+    ) -> Tuple[Union[Monster, NPC], Union[Item, Technique], Monster]:
+        """
+        Trainers battles.
+        """
+        if self.check_strongest(user, monster):
+            if len(user.inventory) > 0:
+                inventory = list(user.inventory)
+                for i in inventory:

```

Comment:
`for item in inventory`
`for item_slug in inventory`

`i` is usually used for indexes

e.g.
`for i in range(10):`

Reply:
item_slug sounds fine